apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1031,7 +1031,12 @@ def to_h5ad(
 
 # ----------------------------------------------------------------
 def to_anndata(
-    experiment: Experiment, measurement_name: str, X_layer_name: str = "data"
+    experiment: Experiment,
+    measurement_name: str,
+    *,
+    obs_id_name: str = "obs_id",
+    var_id_name: str = "var_id",
```

Comment:
In general, for any arbitrary SOMA collection/experiment, these will _not_ work, right? If that's the case, maybe the default should just be `soma_joinid`? Or make it a required param, with no default.

Reply:
No, these are _distinct_ from `soma_joinid`.

The `soma_joinid` column is integers.

What we need here is strings.

Reply:
OK, fair enough!  Still, if obs/var_id is not generally going to work, should we make it a require param w/o a default?

Reply:
@atolopko-czi  I'd rather keep it the default as it's often enough correct (for anything ingested from AnnData, which is very often the case) -- and people can override the default when it's not the case